src/main/java/fi/metatavu/metaform/server/rest/translate/MetaformTranslator.java

```diff
@@ -47,6 +47,10 @@ public Metaform translateMetaform(fi.metatavu.metaform.server.persistence.model.
       result.setId(entity.getId());
     }
     
+    if (entity.getExportTheme() != null) {
+      result.setExportThemeId(entity.getExportTheme().getId());
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "result" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=49&issues=AWWBxMGOXKENQn1S5jLS&open=AWWBxMGOXKENQn1S5jLS)

[ISSUE_KEY]: AWWBxMGOXKENQn1S5jLS